File: data/java/MAXIMUM_SUM_2_X_N_GRID_NO_TWO_ELEMENTS_ADJACENT_TRANSLATED.py
====================
def f_gold ( grid , n ) :
    incl = max ( grid [ 0 ] [ 0 ] , grid [ 1 ] [ 0 ] )
    excl , excl_new = 0 , 0
    for i in range ( 1 , n ) :
        excl_new = max ( excl , incl )
        incl , excl_new = excl + max ( grid [ 0 ] [ i ] , grid [ 1 ] [ i ] )
        excl , excl_new = excl_new , excl_new
    return max ( excl , incl ) , max ( excl_new , incl )

File: data/python/MAXIMUM_SUM_2_X_N_GRID_NO_TWO_ELEMENTS_ADJACENT_TRANSLATED.java
====================
public static int fGold ( int [ ] [ ] grid , int n ) {
  int incl = Math . max ( grid [ 0 ] [ 0 ] , grid [ 1 ] [ 0 ] ) ;
  int excl = 0 ;
  for ( int i = 1 ;
  i < n ;
  i ++ ) {
    int exclNew = Math . max ( excl , incl ) ;
    incl = excl + Math . max ( grid [ 0 ] [ i ] , grid [ 1 ] [ i ] ) ;
    excl = exclNew ;
  }
  return Math . max ( excl , incl ) ;
}

